ibm_catalog.json

```diff
@@ -127,11 +127,28 @@
               }
             },
             {
-              "key": "name"
+              "key": "provider_visibility",
+              "options": [
+                {
+                  "displayname": "private",
+                  "value": "private"
+                },
+                {
+                  "displayname": "public",
+                  "value": "public"
+                },
+                {
+                  "displayname": "public-and-private",
+                  "value": "public-and-private"
+                }
+              ]
+            },
+            {
+              "key": "watsonx_data_instance_name"
             },
 
             {
-              "key": "plan",
+              "key": "watsonx_data_plan",
```

Comment:
We don't seem to be consistent here with the plan input name. In Secrets Manager and Event Notifications you have used `service_plan`. In SCC DA you have used `scc_service_plan`, but thats because it supports creating a second service in that DA (`scc_workload_protection_service_plan`).